src/IssueViz.Security/Taint/ServerSentEvents/ITaintServerEventSource.cs

```diff
@@ -18,9 +18,10 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using SonarLint.VisualStudio.Core.ServerSentEvents;
 using SonarQube.Client.Models.ServerSentEvents.ClientContract;
 
-namespace SonarLint.VisualStudio.Core.ServerSentEvents.TaintVulnerabilities
+namespace SonarLint.VisualStudio.IssueVisualization.Security.Taint.ServerSentEvents
```

Comment:
I moved this to IssueViz.Security assembly and it caused some changes... I should've done it in a separate PR 🙈

Reply:
It seems odd that `ConnectedMode` needs to know about `IssueViz.Security`.

Would it make sense to invert the dependency by moving the interface only to `ConnectedMode`?  The interface is specific to connected mode, so it seems more natural for it to be defined there.

The implementation is visualisation-specific, so that belongs in `IssueViz.Security`